internal/pkg/scorecard/scorecard.go

```diff
@@ -310,7 +314,59 @@ func runTests() ([]TestSuite, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to merge test suite results: %v", err)
 	}
-	return suites, nil
+	for _, suite := range suites {
+		// convert to ScorecardOutput format
+		// will add log when basic and olm tests are separated into plugins
+		pluginResults = append(pluginResults, TestSuitesToScorecardOutput([]TestSuite{suite}, ""))
+	}
+	// Run plugins
+	pluginDir := viper.GetString(PluginDirOpt)
+	if dir, err := os.Stat(pluginDir); err != nil || !dir.IsDir() {
+		log.Warnf("Plugin directory not found; skipping plugin tests")
```

Comment:
Return the result here so we can drop the `else` block. It helps with readability if we fall through.
```Go
if dir, err := os.Stat(pluginDir); err != nil || !dir.IsDir() {
	log.Warnf("Plugin directory not found; skipping plugin tests")
	return pluginResults, nil
}

if err := os.Chdir(pluginDir); err != nil {
...
```